ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credentials/new.js

```diff
@@ -29,15 +29,19 @@ export default class ScopesScopeCredentialStoresCredentialStoreCredentialsNewRou
       'scopes.scope.credential-stores.credential-store'
     );
     let name, description;
+    const json_object = '';
```

Comment:
```suggestion
    const json_object = '{}';
```
This should be set to '{}' so the user isn't shown a linter error right out of the gate.

Reply:
I understand we are doing this for linting purposes, but is quite confusing to set a variable with the word `object` on it as a string.

Linter is a tool to help developers, perhaps I am missing context but I would preffer to skip the linting issue here rather than setting an object as a string.

As a quick test, I checkout this branch, set `const json_object = {}` and got no errors ?